tgui/packages/tgui/styles/main.scss

```diff
@@ -61,6 +61,7 @@
 @include meta.load-css('./interfaces/OreRedemption.scss');
 @include meta.load-css('./interfaces/PanDEMIC.scss');
 @include meta.load-css('./interfaces/PDA.scss');
+@include meta.load-css('./interfaces/pda_minesweeper.scss');
```

Comment:
Also maybe should be separated somehow, maybe in code position

Reply:
don`t see any need in this

Reply:
if we are getting entire PDA sub category for Games, then yeah, I think putting it in something like 
`./interfaces/pda-games/minesweeper.scss` would be nice